src/popup/components/FixedScreenFooter.vue

```diff
@@ -17,6 +18,7 @@ import { IS_MOBILE_DEVICE } from '@/constants';
 export default defineComponent({
   props: {
     noPaddingTop: Boolean,
+    positionFixed: Boolean,
```

Comment:
The initial idea of this component was to be fixed whenever used. I think having additional property that fixes it even more.